admin/app/styles/components/complementary-certifications/attach-target-profile.scss

```
@@ -14,6 +14,18 @@
     margin-bottom: $pix-spacing-l;
   }
 
+  &__notification {
+    display: flex;
+
+    &__checkbox {
+      margin-right: 8px;
+    }
+
+    .pix-tooltip__trigger-element{
+      margin: 0 8px;
+    }
+  }
```

Pour utiliser le design system

```suggestion
    &__checkbox {
      margin-right: $pix-spacing-xs;
    }

    .pix-tooltip__trigger-element{
      margin: 0 $pix-spacing-xs;
    }
  }
```